File: Shared/Resources/bubblesort.bundle/bubblesort.swift
func sort(_ arr: inout [Int]) {
	let n = arr.count
	for c in 0..<(n - 1) {
		for d in 0..<(n - c - 1) {
			if arr[d] > arr[d + 1] {
				arr.swapAt(d, d + 1)
			}
		}
	}
}

var items: [Int] = [35, 95, 74, 71, 72, 30, 96, 53, 9, 0]
sort(&items)
print(items)

File: Shared/Resources/bubblesort.bundle/bubblesort.go
package main

import (
  "fmt"
)

func sort(arr []int) []int {
  n := len(arr)
  for c := 0; c < n - 1; c++ {
    for d := 0; d < n - c - 1; d++ {
      if arr[d] > arr[d + 1] {
        arr[d], arr[d + 1] = arr[d + 1], arr[d]
      }
    }
  }
  return arr
}

func main() {
  array := []int{0, 39, 21, 62, 91, 77, 14, 23,
    90, 69, 51, 81, 68, 83, 32, 56}
  fmt.Println(sort(array))
}